static/src/javascripts/projects/common/modules/crosswords/hidden-input.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import { createClass, createElement, findDOMNode } from 'react/addons';
+import bonzo from 'bonzo';
+import fastdom from 'fastdom';
+import { scrollTo } from 'lib/scroller';
+import { isBreakpoint } from 'lib/detect';
+
+const HiddenInput = createClass({
```

Comment:
Would it be more idiomatic to create this as an ES6 class extending `React.Component`?

Reply:
Think you're right. The convention for the rest of the crossword files is pretty mixed as well though - there's a follow-up project after this to convert crosswords to preact where we can address that across the board